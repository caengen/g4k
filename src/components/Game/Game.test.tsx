import React from "react";
import { shallow } from "enzyme";
import { Game } from ".";

it("renders correctly", () => {
  const wrapper = shallow(<Game />);
  expect(wrapper).toMatchSnapshot();
});
