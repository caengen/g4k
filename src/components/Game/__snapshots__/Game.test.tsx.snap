// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<styled.div>
  <styled.section>
    <styled.h1>
      Kahoot! Points
    </styled.h1>
    <ItemBoard
      items={Array []}
      onClickItem={[Function]}
    />
  </styled.section>
  <styled.section>
    <styled.h1>
      Player items
    </styled.h1>
    <Scoreboard
      items={Array []}
      onClickNewGame={[Function]}
    />
  </styled.section>
</styled.div>
`;
